handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -2,10 +2,6 @@
 
 We are looking for frontend engineers who know how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
```

Comment:
Just delete this file if no teams are hiring for this anymore?

Reply:
There are old blog post entries linking to it.

Reply:
Can we delete the relevant links from those blog posts? Otherwise, we can/should still delete this (to avoid confusion) and @aileenrose can setup redirects to our careers page.